src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -88,7 +95,7 @@ define 'orifice-calculator-viewmodel', ['knockout', 'lodash'], (ko, _) ->
         _.ceil (1 / Math.sqrt(1 - @betaRatio() ** 4)), 2
 
       @flowRate = ko.computed =>
-        operatingTemperatureInRankine = +@operatingTemperature() + ABSOLUTE_ZERO
+        operatingTemperatureInRankine = @operatingTemperature() + ABSOLUTE_ZERO
 
         flowRate = UNIT_CONVERSION_FACTOR * COEFFICIENT_OF_DISCHARGE * EXPANSION_FACTOR * @velocityOfApproach() * @orificeBoreDiameter() ** 2 * BASE_TEMPERATURE/BASE_PRESSURE \
```

Comment:
We will consider breaking this into smaller parts of the operation, instead of a big sequence of values, but not for now